JitsiConference.js

```diff
@@ -2257,22 +2268,23 @@ JitsiConference.prototype._onIceConnectionEstablished = function(
         logger.info('Not removing remote JVB tracks - no session yet');
     }
 
-    // Add remote tracks
     this._addRemoteP2PTracks();
 
     // Stop media transfer over the JVB connection
     if (this.jvbJingleSession) {
         this._suspendMediaTransferForJvbConnection();
     }
 
-    // Start remote stats
     logger.info('Starting remote stats with p2p connection');
     this.statistics.startRemoteStats(this.p2pJingleSession.peerconnection);
 
-    // Log the P2P established event
-    if (this.p2pJingleSession.isInitiator) {
-        Statistics.sendEventToAll(P2P_ESTABLISHED);
-    }
+    Statistics.sendAnalyticsAndLog(
```

Comment:
@virtuacoplenny brought up a point when we were talking yesterday...do we ever want to fire an event and not log it?  should logging an event just be the default?

Reply:
(that being said, this name is probably still good...just wondering if the plain `sendEvent` should exist)

Reply:
Well "logging" in this context refers to sending it to callstats, so we actually rarely want to "log" analytics events.

Reply:
oh, if the `Log` in `sendAnalyticsAndLog` refers to sending to callstats, what does the `sendAnalytics` refer to?

Reply:
Sending to whatever event-based analytics backend has been configured